### PR TITLE
added requirements.txt, try_requests. py, and palindrome_check.py wit…

### PR DESCRIPTION
**Pull request by shammercode**
…h corresponding test_palindrome_check.py

### CODE REVIEW

File: Demos/Module2/palindrome_check.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            string tested for palindrome
+    Returns:
+        bool
+            True if s is a palindrome, False if not
+    """
+    forward_list = []
+    stack = []
+    s=s.lower()
+    for i in s:
+        if i.isalpha():
```
Comment (lagishtein) [inline Demos/Module2/palindrome_check.py]: this will handle special characters but not the numbers. There is another function in python that will handle both

**Review by lagishtein — COMMENTED** [review]
- please fix the function used to keep just the regular characters.
- add unit tests for single character and empty space.
- do some searching and check if there is a better way to calculate a palindrome that will make a more efficient use of memory

File: Demos/Module2/palindrome_check.py (hunk 1 above)
Comment (shammercode) [reply Demos/Module2/palindrome_check.py]: Numbers should be allowed, or not?

Comment (lagishtein) [reply Demos/Module2/palindrome_check.py]: they should be cleaned out in the same fashion as special characters

**Review by lagishtein — APPROVED** [review]
Approved. Please merge
